inference-engine/ie_bridges/python/install/setup.py.in

```diff
@@ -0,0 +1,249 @@
+
+
+class CopyExt(build_ext):
+    def run(self):
+        for extension in self.extensions:
+            if not isinstance(extension, PrebuiltExtension):
+                raise DistutilsSetupError(
+                    "copy_ext can accept PrebuiltExtension only")
+            src = extension.sources[0]
+            dst = self.get_ext_fullpath(extension.name)
+            try:
+                os.makedirs(os.path.dirname(dst), exist_ok=True)
+            except OSError as exc:
+                if exc.errno != errno.EEXIST:
+                    raise
+
+            #setting relative path to find dlls
+            if sys.platform != "win32":
+                rpath = os.path.relpath(BINDINGS_DIR, os.path.dirname(src))
+                if sys.platform == "linux":
+                    rpath = os.path.join("$ORIGIN", rpath, WHEEL_LIBS_INSTALL_DIR)
+                elif sys.platform == "darwin":
+                    rpath = os.path.join("@loader_path", rpath, WHEEL_LIBS_INSTALL_DIR)
+                set_rpath(rpath, src)
+
+            copy_file(src, dst, verbose=self.verbose, dry_run=self.dry_run)
+
+def set_rpath(rpath, executable):
+    print(f"Setting rpath {rpath} for {executable}")
+    cmd = ""
+    if sys.platform == "linux":
+        cmd = ["patchelf", "--set-rpath", rpath,  executable]
+    elif sys.platform == "darwin":
+        cmd = ["install_name_tool", "-add_rpath", rpath,  executable]
+    ret_info = subprocess.run(cmd)
+    if ret_info.returncode != 0:
+        print(f"The exit code: {ret_info.returncode}")
+        print(f"Could not set rpath: {rpath} for {executable}")
+        exit(2)
+
+
+def find_data_files(src_dirs, install_subdir):
+    data_files = []
+    # excluding unneeded libs for wheel distribution
+    data_blacklist = ['.lib', '.pdb', '_debug.dll']
+    for dir in src_dirs:
+        local_base_dir = Path(dir)
+        for root, directories, filenames in os.walk(local_base_dir):
+            for filename in filenames:
+                if not any(filename.endswith(ext) for ext in data_blacklist):
+                    data_files.append([os.path.join(install_subdir, os.path.relpath(root, local_base_dir)), [os.path.join(root, filename)]])
+
+    return list(data_files)
+
+
+def find_prebuilt_extensions(base_dir, ext_pattern):
+    extensions = []
+    for path in Path(base_dir).glob(ext_pattern):
+        relpath = path.relative_to(base_dir)
+        if relpath.parent != ".":
+            package_names = str(relpath.parent).split(os.path.sep)
+        else:
+            package_names = []
+        package_names.append(path.name.split(".", 1)[0])
+        name = ".".join(package_names)
+        extensions.append(PrebuiltExtension(name, sources=[str(path)]))
+    return extensions
+
+
+# reading description from README.md
+with open("pypi_overview.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+# reading dependencies from requirements.txt
+with open("requirements.txt", "r", encoding="utf-8") as fh:
```

Comment:
can be absent in a current dir, use full path

Reply:
It should be in the current directory, which is expected. if not then the user has to define the right path in the script or put the file in the right place.